pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -568,6 +568,27 @@
             "#import": [ "opensuse.42.1", "opensuse-x64" ]
         },
 
+        "gentoo": {
+            "#import": [ "linux" ]
+        },
+        "gentoo-x64": {
+            "#import": [ "gentoo", "linux-x64" ]
+        },
+
+        "buildroot": {
+            "#import": [ "linux" ]
+        },
+        "buildroot-x64": {
+            "#import": [ "buildroot", "linux-x64" ]
+        },
+
+        "buildroot.2017.02.2": {
+            "#import": [ "buildroot" ]
+        },
+        "buildroot.2017.02.2-x64": {
```

Comment:
There is a new release twice a year, and the incremental small fix releases.

I just added the latest current release - if the last segment can be ignored (not sure if it can match it if it is changed to 2017.02-x64